dev-java/abcl/abcl-1.9.2.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A Java-based common lisp implementation"
+DESCRIPTION="Armed Bear Common Lisp (ABCL) is a full implementation \
+of the Common Lisp language featuring both an interpreter and a compiler, \
+running in the JVM. Originally started to be a scripting language for \
+the J editor, it now supports JSR-223 (Java scripting API): it can be a \
+scripting engine in any Java application. Additionally, it can be used \
+to implement (parts of) the application using Java to Lisp integration APIs."
+HOMEPAGE="http://abcl.org/"
+COPYRIGHT="1999-2023 ABCL development team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://abcl.org/releases/$portVersion/abcl-bin-$portVersion.tar.gz"
+CHECKSUM_SHA256="24970976b3565ddf32a1e0b17c5034a9996df25404ec44f240505b01c68a37fe"
+SOURCE_DIR="abcl-bin-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	abcl = $portVersion compat >= 1
+	cmd:abcl = $portVersion compat >= 1
+	"
+REQUIRES="
+	haiku
+	java:environment >= 17
+	"
+
+BUILD_REQUIRES="
+	"
+BUILD_PREREQUIRES="
+	java:environment == 17
+	"
+
+INSTALL()
+{
+	source /system/data/profile.d/openjdk17.sh
+	export JAVA_HOME=$JDK17_HOME
```

Comment:
Why is this needed?

Reply:
i am not entirely sure. it came from the earlier port definitions where i might have ran into some circular conditions that don't appear to exist anymore.  have removed this. thanks!